datafusion/sqllogictest/test_files/select.slt

```diff
@@ -483,6 +483,17 @@ CREATE TABLE foo2(c1 double, c2 double) AS VALUES
 (2, 5),
 (3, 6);
 
+statement ok
+SELECT COALESCE(column1, column2) FROM VALUES (null, 1.2);
+
+# multiple rows and columns with null need type coercion
+statement ok
+SELECT column1, column2, column3 FROM VALUES
```

Comment:
We can try subquery to show the type

`select * from (select * from values (...))`